docs/user/experiments.rst

```diff
@@ -74,7 +74,16 @@ simple one, :code:`examples/tf/trpo_cartpole.py`, is also pasted below:
 
   trpo_cartpole()
 
+.. testoutput::
+   :hide:
+   
+   ...
 
+.. testcleanup::
+	
+	import shutil
+	shutil.rmtree('data/local/')
```

Comment:
is there any way to do this session-wide rather than in every test?

Reply:
I could put this into `make doctest`